public/app/features/datasources/constants.ts

```diff
@@ -10,3 +10,15 @@ export const DATASOURCES_ROUTES: DataSourcesRoutes = {
   Dashboards: '/datasources/edit/:uid/dashboards',
   New: '/datasources/new',
 } as const;
+
+/**
+ * Plugin IDs that are allowed to contribute extensions to datasource configuration pages
```

Comment:
I think this could be a bit misleading, as the `EditDatasource` page allows only `grafana-pdc-app` and `grafana-auth-app`: https://github.com/grafana/grafana/blob/main/public/app/features/datasources/components/EditDataSource.tsx#L271